pkg/apiserver/apiserver.go

```diff
@@ -341,6 +351,7 @@ func AddSupportedResourcesWebService(s runtime.NegotiatedSerializer, ws *restful
 		ss = StripVersionNegotiatedSerializer{s}
 	}
 	resourceHandler := SupportedResourcesHandler(ss, groupVersion, apiResources)
+	ws.SetDynamicRoutes(true)
```

Comment:
Would be good to add a comment saying why are we setting this.
I assume it is so that we can call RemoveRoute()?